pyrobosim/pyrobosim/utils/pose.py

```diff
@@ -64,7 +86,12 @@ def from_list(cls, plist):
         elif num_elems == 3:
             return cls(x=plist[0], y=plist[1], z=plist[2])
         elif num_elems == 4:
-            return cls(x=plist[0], y=plist[1], z=plist[2], yaw=plist[3])
+            return cls(
```

Comment:
Maybe worth leaving a comment in the docstring of this function that the angle units are always radians for this function. Just to be safe.